frontend/internal-packages/agent/src/chat/workflow/nodes/designSchemaNode.ts

```diff
@@ -226,28 +108,19 @@ Please fix this issue by analyzing the schema and adding any missing constraints
     }
   }
 
-  const result = await handleSchemaChanges(
-    invokeResult.value,
-    buildingSchemaVersionId,
-    state,
-    repositories,
-    assistantRole,
-  )
-
   await logAssistantMessage(
     state,
     repositories,
     'Schema design completed',
     assistantRole,
   )
 
-  // Clear retry flags after processing
-  const finalResult = {
-    ...result,
-    messages: [...messages, invokeResult.value.message],
+  return {
+    ...state,
+    messages: [invokeResult.value],
+    buildingSchemaVersionId: createVersionResult.versionId,
+    latestVersionNumber: state.latestVersionNumber + 1,
```

Comment:
Because this area was not incremented, it seemed to cause a conflict error on the second and subsequent user entries.